_events/2023-01-25-htcondor-osg.md

```diff
@@ -43,7 +46,9 @@ View last year’s schedules for
 - [HTCondor Week 2022](https://agenda.hep.wisc.edu/event/1733/timetable/#20220523)
 
 {% endcapture %}
+### What
```

Comment:
From here down seems entirely repetitive.

Reply:
Except for the Who paragraph. Maybe just move that up and delete the repetitive parts?

Reply:
If you are talking about the Where, Who etc. this is repetitive - it is a different presentation of much of the same info intended for those that do not read paragraphs of text. I'm going to leave it - We have been using this format for a while.